fe/fe-core/src/main/java/com/starrocks/clone/TabletSchedCtx.java

```diff
@@ -495,13 +495,16 @@ public boolean isDestPathResourceHold() {
         return this.destPathResourceHold;
     }
 
-    // database lock should be held.
-    public void chooseSrcReplica(Map<Long, PathSlot> backendsWorkingSlots) throws SchedException {
-        /*
-         * get all candidate source replicas
-         * 1. source replica should be healthy.
-         * 2. slot of this source replica is available.
-         */
+    public boolean needCloneFromSource() {
+        return tabletStatus == TabletStatus.REPLICA_MISSING ||
+                tabletStatus == TabletStatus.VERSION_INCOMPLETE ||
+                tabletStatus == TabletStatus.REPLICA_RELOCATING ||
+                tabletStatus == TabletStatus.FORCE_REDUNDANT ||
```

Comment:
`FORCE_REDUNDANT` does not need to check source replica health.

Reply:
Yes, `FORCE_REDUNDANT` is just like `REDUNDANT`, only delete one replica, but in force mode, and let the repair to be done later. I will remove this condition.